crowbar_framework/spec/models/api/crowbar_spec.rb

```diff
@@ -125,6 +125,45 @@
     end
   end
 
+  context "with ceph cluster healthy" do
+    it "succeeds to check ceph cluster health" do
+      allow(NodeObject).to(
+        receive(:find).with("roles:ceph-mon AND ceph_config_environment:*").
+        and_return([NodeObject.find_node_by_name("testing.crowbar.com")])
+      )
+      allow_any_instance_of(NodeObject).to(
+        receive(:run_ssh_cmd).with("LANG=C ceph health 2>&1").
+        and_return(exit_code: 0, stdout: "HEALTH_OK\n", stderr: "")
+      )
+      expect(subject.ceph_healthy?).to be true
+    end
+  end
+
+  context "with ceph cluster not healthy" do
+    it "fails when checking ceph cluster health" do
+      allow(NodeObject).to(
+        receive(:find).with("roles:ceph-mon AND ceph_config_environment:*").
+        and_return([NodeObject.find_node_by_name("testing.crowbar.com")])
+      )
+      allow_any_instance_of(NodeObject).to(
+        receive(:run_ssh_cmd).with("LANG=C ceph health 2>&1").
+        and_return(exit_code: 1, stdout: "Some message", stderr: "")
```

Comment:
not that it makes a real difference in the test, but

```ruby
and_return(exit_code: 1, stdout: "HEALTH_WARN", stderr: "")
```

is more representative for reality